modules/openshift-developer-cli-installing-odo-on-rhel.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * assebmly_installing-odo.adoc     
+
+[id="installing-odo-on-rhel"]
+
+= Fedora, CentOS, or RHEL
+
+== Using yum or dnf
+. Access the `odo` repository:
```

Comment:
This is technically a procedure but doesn't have the normal formatting of a procedure module.

I don't know what the best solution is. It's probably overkill to make each OS's different methods each its own procedure module, but it does throw me off a little as is.